tests/data_structures/linked_list.rs

```diff
@@ -0,0 +1,44 @@
+use rusthms::algorithms::data_structures::linked_list::LinkedList;
+use rusthms::algorithms::data_structures::linked_list::Node;
+
+fn create_node(id: i32) -> Node<i32> {
+    Node {
+        data: Some(Box::new(id)),
+        next: None,
+    }
+}
+
+fn create_linked_list() -> LinkedList<i32> {
+    LinkedList {
+        tail: None,
+        size: 0,
+    }
+}
```

Comment:
@Segmentree these methods should be in the linked_list definition, not in the test, I think, also, it would be cool if we can do `Node<i32>::new()`, instead `create_node()`

Reply:
@Segmentree same with `LinkedList`, I mean, be able to call `LinkedList<i32>::new()`